Traceback (most recent call last):
  File "/home/F3/src/abstract_loops.py", line 292, in gen_loops
    sat = solver.solve()
  File "/home/F3/src/multisolver.py", line 117, in solve
    return self._get_solve_result(res)
  File "/home/F3/src/multisolver.py", line 131, in _get_solve_result
    raise SolverReturnedUnknownResultError()
pysmt.exceptions.SolverReturnedUnknownResultError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/F3/src/run_f3.py", line 444, in <module>
    main(config_args().parse_args())
  File "/home/F3/src/run_f3.py", line 440, in main
    run_f3(bench, out_dir)
  File "/home/F3/src/run_f3.py", line 260, in run_f3
    res = solve_problem(prob, label, out_dir)
  File "/home/F3/src/run_f3.py", line 251, in solve_problem
    return run_ltl_test(problem, env, label, out_dir)
  File "/home/F3/src/run_f3.py", line 113, in run_ltl_test
    return search_nonterm(env, symbols, init, trans, fairness,
  File "/home/F3/src/run_f3.py", line 169, in search_nonterm
    _, arg0, arg1 = search_funnel_bmc(env, symbols, init, trans,
  File "/home/F3/src/main.py", line 155, in search_funnel_bmc
    for i, (trace, lback_idx, abst_path, hints) in \
  File "/home/F3/src/abstract_loops.py", line 303, in gen_loops
    solver.push()
  File "/home/F3/src/multisolver.py", line 145, in push
    s.push(levels=levels)
  File "/usr/local/lib/python3.8/dist-packages/PySMT-0.9.1.dev332-py3.8.egg/pysmt/solvers/solver.py", line 365, in push
    self._push(levels=levels)
  File "/usr/local/lib/python3.8/dist-packages/PySMT-0.9.1.dev332-py3.8.egg/pysmt/decorators.py", line 64, in clear_pending_pop_wrap
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/PySMT-0.9.1.dev332-py3.8.egg/pysmt/solvers/z3.py", line 264, in _push
    self.z3.push()
  File "/usr/local/lib/python3.8/dist-packages/z3/z3.py", line 6529, in push
    Z3_solver_push(self.ctx.ref(), self.solver)
  File "/usr/local/lib/python3.8/dist-packages/z3/z3core.py", line 3742, in Z3_solver_push
    _elems.f(a0, a1)
KeyboardInterrupt
real 3602.43
user 0.00
sys 0.00
